docs/release-notes/v0.6.0.adoc

```diff
@@ -10,3 +10,4 @@
 * Add IT test for owned helpers
 * GET `/system/evict` REST endpoint added
 * Template caching has been fixed
+* Template caching can be configured with the environment variable named `DOOKUG_SERVICE_CACHE_TEMPLATE_ENABLED` (`true` by default) 
```

Comment:
a doksi config reszeben is kellene

Reply:
benne van, csak veletlenul meg az elozo MRben ment be :)

(install.adoc)